apps/mobile/app.json

```diff
@@ -2,11 +2,14 @@
   "expo": {
     "name": "tequila-client",
     "entryPoint": "./apps/mobile/index.js",
-    "description": "This project is really great.",
+    "description": "Margarita is an Open-Source, Multi-platform travel app for all major platforms — one code for web, iOS and Android. More information on https://margarita.kiwi.com/",
```

Comment:
What about this change?

"Margarita is an open source, multi-platform travel app..."

Reply:
Sure!